.travis.yml

```
@@ -1,62 +1,41 @@
 language: python
 python: 3.6.6
```

```suggestion
 python: pypy3.5-6.0

 dist: xenial
```
can enable you run PyPy v6.0 in Travis